src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -51,7 +51,12 @@ public void prepareMessageForDisplay(Message message) {
     String replacement = "<img src=\"$1\" />";
     String text = message.getText();
     text = text.replaceAll(regex, replacement);
+    text = text.replace("[b]", "<strong>").replace("[/b]", "</strong>");
```

Comment:
Since your additions are fairly distinct from the image-linking code, you could put a blank line to separate them, and then add a short comment just to say that your lines handle text styling markup.